src/views/tutorial-view/components/featured-in-collections/index.tsx

```diff
@@ -25,9 +25,10 @@ export function FeaturedInCollections({
 				<CardsGridList fixedColumns={collections.length == 1 ? 2 : null}>
 					{collections.map((cardPropsWithId: CollectionCardPropsWithId) => {
 						return (
-							<li key={cardPropsWithId.id} className={s.listItem}>
```

Comment:
Note: the `s.listItem` class removed here didn't actually exist, so wasn't doing anything.

You can see this by inspecting the "Featured in collections" are of a live page such as [/terraform/tutorials/aws-get-started/install-cli](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) - note that the `<li />` elements do not have a `className` rendering (because here in the code, `s.listItem` is undefined).

Reply:
good catch!